### ISSUE TITLE
Unable to start server, location unrecognized, ClubLogin fails, help needed ) how retry?

### ISSUE THREAD

**Issue by CharliePrown** [AHAAAAAAA/PokemonGo-Map#1832]
Hi guys, sorry for beeing bad with any phyton and commands(

Just found a link to your map tool and a guide, so i got my google api and another pokemonclub login.
Through the pokogo master i ran the easy setup, that psi thing is done, jsan is done, google maps location town to coordinates convertion is done.

So last thing is to use:
phyton runserver.....login...coordinates....command in the cmd window

Doing this with coordinates it gives me an unknwon thing with second part of two part coordinates, why?  Format should be right.

Using a location name instead of coordinates without the brackets works in running, but town name is that often in the world, one it would not be right, second with an additional info to country/region it gives me an error again.

However, as is use just the town name, even it might be the wrong one, club login is stated as wrong even data is right, retry in 15 minutes.

Due to server issues at the club?

Login through their website works.

Well and last, easy setup, runs setup again, how to just start the cmd window for the last command line now again after setup for next try or after booting the system tomorrow?

thx for your help

**Comment by gramthanos** [comment]
Try to set a location on a road.

**Comment by CharliePrown** [comment]
well watched that video, he uses an official hospital and not coordinates but copy and pastes the syntax words, i ll try that.

but how to restart the whole thing now once more please?

do i have to start phyton.exe or what to enter the username...api...location thing once more?

thank you)

**Comment by CharliePrown** [comment]
BTw, from another thread:
python.exe" runserver.py -a ptc -u username -p password -l "myhometown" -st 6 -H 0.0.0.0 -L de -sd 0

Do I need that thing after my hometown in any case? as it isnt shown so in the wiki?

**Comment by CLINTZ9** [comment]
Although this is the syntax for the previous build just have a look at it
`c:\Python27\python example.py -a ptc -u username -p password -l "12.900003, 74.841980" -st 10 `

**Comment by CharliePrown** [comment]
so i have to adress over the phyton folder at any start right?

and is it a problem i put that pogomap downloads in a sperate folder on c, not the phyton installation folder?

**Comment by CLINTZ9** [comment]
No its not like that `c:\Python27\python` is to call the python executable 
You can put the pogomap anywhere

**Comment by CharliePrown** [comment]
well my command from the wiki is with 

python runserver.py -u [USERNAME] -p [PASSWORD] -st 10 -k [Google Maps API key] -l "[LOCATION]"

which gives me the error no....runserver.py...no such fiel or directory, cant open runserver?

and as i use club login data i added -ptc in front of -u for sure?)

**Comment by CLINTZ9** [comment]
Will get back to you soon 
Currently on linux
Will try the latest windows version
Meanwhile add python to your execution path 
Also cross check if your on python 2.7

**Comment by CharliePrown** [comment]
yep 2.7 

thx so far, no hurries

**Comment by CLINTZ9** [comment]
Replace your credentials `c:\Python27\python runserver.py -u USERNAME -p PASSWORD -st 10 -k GMAPSAPIKEY -l "[Valencia,Mangalore]"   `

**Comment by CharliePrown** [comment]
can't open runserver.py no such file, stays the same issue(

**Comment by CLINTZ9** [comment]
Are you in the right directory?

**Comment by CharliePrown** [comment]
https://github.com/AHAAAAAAA/PokemonGo-Map/issues/886

trying this now
well in start the command from c as python is there?

c main tree

**Comment by CharliePrown** [comment]
does not work too, so you said it does not matter, but:
 i have c:python
and i have c:pogomap....where the downloaded master is in where there the server to run is in, so i copy and paste it into python too?

**Comment by CLINTZ9** [comment]
![capture](https://cloud.githubusercontent.com/assets/13290010/17085043/ca8b129a-51eb-11e6-908b-6e4008906a7e.PNG)
![capture1](https://cloud.githubusercontent.com/assets/13290010/17085044/cab97220-51eb-11e6-9c29-b162e6f0bbeb.PNG)

**Comment by CharliePrown** [comment]
As I cd to c main, then cd to Python 27 and then adress the command it works

But once more with correct PTC Data i get a login failure and am locked for 15 minutes

Using the username and password without the [ ]

**Comment by CharliePrown** [comment]
SORRY, copied wrong, used brakcets, now without i am in.)

Just need to check if he used right town or else modify coordinates a bit.

THX a lot.)))

**Comment by CLINTZ9** [comment]
you dont need to cd to python27
stay in the master zip directory 
Look closely at the second screenshot

**Comment by CLINTZ9** [comment]
Your welcome :)
Happy pokemon hunting :dancers: 
You can close the issue now :+1:

**Comment by CharliePrown** [comment]
Nope, as scan stays in loop, so he scans in background all the time i guess.

LEaving it running and doing the localhost:5000 thing connection still gets refused.)

**Comment by CLINTZ9** [comment]
Send a screenshot of the command prompt  and the
Browser

**Comment by CharliePrown** [comment]
So he circled several times from 1-271, started new from 1 and now is stuck at 3x,xx percent.


![cmd window](https://cloud.githubusercontent.com/assets/20626159/17085139/70ebb98e-51d1-11e6-80ed-6ae221f56855.jpg)


![browser](https://cloud.githubusercontent.com/assets/20626159/17085140/7337d10a-51d1-11e6-9816-dbb0e1005e7f.jpg)

**Comment by CLINTZ9** [comment]
try http://127.0.0.1:5000/

**Comment by CharliePrown** [comment]
THX :dancers:

**Comment by CLINTZ9** [comment]
done?

**Comment by CharliePrown** [comment]
ok, deleted, thx.

**Comment by CLINTZ9** [comment]
ok then good night :dancers:
